packages/cli/src/lib/implementation/filter.middleware.ts

```diff
@@ -141,3 +163,40 @@ function filterPluginsFromCategories({
   );
   return plugins.filter(plugin => validPluginSlugs.has(plugin.slug));
 }
+
+function filterSkippedItems<T extends { isSkipped?: boolean }>(
+  items: T[] | undefined,
+): Omit<T, 'isSkipped'>[] {
+  return (items ?? [])
+    .filter(({ isSkipped }) => isSkipped !== true)
+    .map(({ isSkipped, ...props }) => props);
+}
+
+export function processPlugins(plugins: PluginConfig[]): PluginConfig[] {
```

Comment:
The name `processPlugins` isn't very self-explanatory:

```suggestion
export function filterSkippedInPlugins(plugins: PluginConfig[]): PluginConfig[] {
```